packages/contract/lib/reason.js

```diff
@@ -23,14 +29,28 @@ const reason = {
       res && typeof res === "object" && typeof res.result === "string";
 
     if (isObject) {
+      // TODO: ganache 2.0 returns the reason string already when
```

Comment:
You've left a `TODO` comment in your code, @gnidan isn't going to like that. :P  So, uh, you should probably figure out whether you want to include that in this PR, or just convert this to a non-TODO comment about how this would be a good thing to do in the future.